embedded/appendable/remoteapp/options.go

```diff
@@ -26,20 +26,18 @@ type Options struct {
 	multiapp.Options
 	parallelUploads int
 
-	retryMinDelay    time.Duration
-	retryMaxDelay    time.Duration
-	retryDelayExp    float64
-	retryDelayJitter float64
+	retryMinDelay time.Duration
+	retryMaxDelay time.Duration
+	retryDelayExp float64
 }
 
 func DefaultOptions() *Options {
 	return &Options{
-		Options:          *multiapp.DefaultOptions(),
-		parallelUploads:  10,
-		retryMinDelay:    time.Second,
-		retryMaxDelay:    2 * time.Minute,
-		retryDelayExp:    2,
-		retryDelayJitter: 0.1,
```

Comment:
We're removing this jitter argument - is it replaced by something else?

Reply:
I realised it wasn't needed, but have added it back to keep the jitter logic the same as you'd previously written

Reply:
fixed